packages/pds/package.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/bluesky-social/atproto.git",
     "directory": "packages/pds"
   },
-  "main": "src/index.ts",
+  "main": "dist/index.js",
```

Comment:
intentional fix, or accidental commit?

Reply:
What's even up with that, when I do `yarn install` is does it automagically

Reply:
accidental

yeah I'm honestly not sure where it comes from. has bitten us a couple times